src/FSharpPlus/Operators.fs

```diff
@@ -93,10 +93,10 @@ module Operators =
     // Un-zips (un-tuple) two functors.
     let inline unzip (source: '``Functor<'T1 * 'T2>``) = Unzip.Invoke source : '``Functor<'T1>`` * '``Functor<'T2>``
 
-    // Zips (tuple) two functors.
+    // Zips safely two collections. If one collection is shorter, excess elements are discarded from the right end of the longer collection. 
```

Comment:
With this comment you are downgrading a function which operates with 2 functors to a function which operates only with 2 collection functors :)  

I see your intention of clarifying the "shortest" behavior and I share it.

Maybe, leave the original comment and add yours in remarks, something like "for collections, if one collection is shorter ..."

Reply:
Changed as suggested. :-)